operator/pkg/securedcluster/scanner/auto_sense_test.go

```diff
@@ -53,3 +63,34 @@ func TestAutoSenseIsEnabledWithCentralInADifferentNamespace(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, enabled, "Expected Scanner to be enabled if Central is deployed in a different namespace")
 }
+
+func TestAutoSenseIsDisabledIfClusterVersionNotFound(t *testing.T) {
+	client := testutils.NewFakeClientBuilder(t, &openshiftConfigv1.ClusterVersion{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "not-version-name",
+		},
+		Spec: openshiftConfigv1.ClusterVersionSpec{
+			ClusterID: "test-cluster-id",
+		},
+	}).Build()
+
+	enabled, err := AutoSenseLocalScannerSupport(context.Background(), client, securedCluster)
+	require.Error(t, err)
+	require.False(t, enabled, `Expected an error if clusterversions.config.openshift.io %q not found`, clusterVersionDefaultName)
```

Comment:
Why backticks?

Reply:
There were "" in the comment but you are right, it is no longer need for `backticks`